front/app/modules/commercial/user_custom_fields/services/stats.ts

```diff
@@ -121,28 +121,3 @@ export function usersByDomicileStream(
     ...streamParams,
   });
 }
-
-export function votesByGenderStream(streamParams: IStreamParams | null = null) {
```

Comment:
@adessy heads up! These endpoints are not used anymore by the FE - should we delete them?

Reply:
Is it related to the clustering engine? Otherwise, I'd rather log this into Jira for a future PR.

Reply:
It was used in the clustering engine on the FE only but I don't have enough context to know whether it's strictly related 🤷

Reply:
Yes, my first thought was that this does not look obviously linked to whatever the clusterings engine was (though it might be).